src/core/LoadBalancer.ts

```diff
@@ -24,13 +24,12 @@ import {
 } from './MembershipListener';
 
 /**
- * {@link LoadBalancer} allows you to send operations to one of a number of endpoints (Members).
+ * A {@link LoadBalancer} allows you to send operations to one of a number of endpoints (Members).
  * It is up to the implementation to use different load balancing policies.
  *
- * If the client is configured with {@link ClientNetworkConfig.smartRouting},
- * only the operations that are not key based will be routed to the endpoint
- * returned by the LoadBalancer. If it is not, the LoadBalancer will not be used.
- *
+ * If the client is a smart client configured with {@link ClientNetworkConfig.smartRouting},
+ * only the operations that are not key based will be routed via LoadBalancer.
+ * If the client is not a smart client, the LoadBalancer will not be used.
```

Comment:
How about: "The LoadBalancer is used only when the client is in the smart mode and an operation is not directed to a specific endpoint."

Reply:
Again, can a user direct an operation to a specific member? I dont know. So let's hide that detail from the user. Which part do you propose to change by the way? Only the  "If the client is not a smart client," sentence?

Reply:
The user obviously cannot direct an operation to a specific member, but if the operation (invocation) is already directed to a key or member (PNCounter) then hte load balancer is not used. I think "only the operations that are not key based will be routed via LoadBalancer" has at least the same implementation detail with "The LoadBalancer is used only when the client is in the smart mode and an operation is not directed to a specific endpoint." (if not more).

I propose to replace lines 30-32 (the lines I've highlighted) with  "The LoadBalancer is used only when the client is in the smart mode and an operation is not directed to a specific endpoint." which I think summarizes those lines.

Reply:
Hmm, ok but I think it is better to explain which operations are directed to a specific endpoint.(key based operations). So the current version gives more info to user. 

I realized that this info is important for user and do not have my opinion I have in the past anymore:  
>  I think "when the operation is not directed to a specific endpoint." is internal detail and user does not need to know it.

Reply:
Key based operations are not the only ones exempt from the load balancer, member based ones are also exempt. Also if an operation has a specific target (connection) then the load balancer will not be used. So load balancer is only used when a random connection is needed.

Reply:
Hmm what about writing that instead